netdata/netdata
# Migrate metadata log to SQLite

##### Summary
Migrate metadata to SQLite

##### Component Name
database

##### Test Plan
- Use this PR on an agent with metadata log support
  - A new database file `netdata-meta.db` will be created under the default cache directory `var/cache/netdata`

- Use cases
  - Parent agent with no metadata support (version prior to 1.22)
    - metrics should be available after migration
  - Parent agent with metadata support (version 1.22 and later)
    - metrics should be available after migration
  - Parent agent any version
    - Children with  (version prior to 1.22)
    - Children with  (version 1.22 and later)
